app/views/ontologies/sections/_metadata.html.haml

```diff
@@ -11,7 +11,8 @@
       = ontology_depiction_card
       = properties_card(t('ontologies.sections.identifiers'),t("ontologies.sections.identifiers"), @identifiers) do |values|
         = horizontal_list_container(values) do |v|
-          = render LinkFieldComponent.new(value: v, check_resolvability: true)
+          - generate_htaccess = v.eql?(@identifiers["ontology_portal_uri"].first)
```

Comment:
do `@identifiers["ontology_portal_uri"]&.first` (added `&`) otherwise it will raise nil exception when not set.